### DISCUSSION TITLE
refresh parent model on child model submit

### DISCUSSION THREAD

**Discussion by StaffCollab** [filamentphp/filament#8276] [Help]
I have a model that processes a billing query, and if the user wants they can add a new payment method (which is a child form on that action).
How can I make the parent model (or action) reflect the updated payment method after the child model is submitted?

**Comment by leandrocfe** [comment]
using [livewire events](https://stackoverflow.com/questions/66094788/laravel-livewire-how-to-force-parent-component-refresh)?

**Reply by StaffCollab** [reply]
You might be right, but I never figured out how to use event listeners in a filament form (not a full-form component but within a form in an action that resides in a different filament page).
Can you help me? is there a listen method on a form component? or so?

And I cannot refresh the full component as it will close the model that the user is filling out... we just need to refresh one form component, how on earth are we doing that?

**Reply by leandrocfe** [reply]
Maybe you can share the code you are trying to do
